
syntax = "proto3";

package Cloud;

service protokol_1 {
  rpc initial_data (pesan_client) returns(pesan_server){}
}

message pesan_client {
  string header_pesan = 1;
  repeated string aset = 2;
  repeated string rute = 3;
  repeated int32 id_param_lama = 4;
  repeated int32 id_tipe_param = 5;
  repeated int32 id_rute_lama = 6;
  repeated int32 timestamp = 7;
  repeated int32 siklus = 8;
  int32 jumlah_data = 9;
}

message pesan_server {
  string header_pesan = 1;
  repeated string aset = 2;
  repeated string rute = 3;
  repeated int32 id_param_lama = 4;
  repeated int32 id_tipe_param = 5;
  repeated int32 id_rute_lama = 6;
  repeated int32 timestamp = 7;
  repeated int32 siklus = 8;
  int32 jumlah_data = 9;
}

